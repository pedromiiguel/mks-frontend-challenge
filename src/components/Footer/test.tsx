import { customRender } from '@/utils/tests/customRender';
import { screen } from '@testing-library/react';

import { Footer } from '.';

describe('<Footer />', () => {
  it('should render the footer', () => {
    const { container } = customRender(<Footer />);

    expect(
      screen.getByText(/MKS sistemas Â© Todos os direitos reservados/i)
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
