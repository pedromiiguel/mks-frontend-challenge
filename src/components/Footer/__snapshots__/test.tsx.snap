// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the footer 1`] = `
.c0 {
  background: #eeeeee;
  padding-block: 1.6rem;
  margin-top: 8rem;
}

.c1 {
  font-weight: 400;
  font-size: 1.2rem;
  line-height: 1.5rem;
  color: #000000;
  text-align: center;
}

<footer
  class="c0"
>
  <p
    class="c1"
  >
    MKS sistemas Â© Todos os direitos reservados
  </p>
</footer>
`;
